Sources/MockoloFramework/Utils/TypeParser.swift

```diff
@@ -566,7 +566,7 @@ public final class SwiftType {
 
         let suffixStr = [
             isAsync ? String.async + " " : nil,
-            throwing.hasError ? String.throws + " " : nil,
+            throwing != .none ? throwing.applyThrowingTemplate()! + " " : nil,
         ].compactMap { $0 }.joined()
 
```

Comment:
It want to avoid relying on internal implementations.

```swift
            let suffixStr = [
                isAsync ? String.async : nil,
                throwing.applyThrowingTemplate(),
            ].compactMap { $0 }.joined(separator: " ") + " "
```

(This snippet exists in the other 2 places. Maybe it's a time to add utility)

Reply:
Sounds nice!  I fixed it in these 2 commits.

- [be998e1](https://github.com/uber/mockolo/pull/262/commits/be998e110e8c09a71d1917b5a01102675db78f74)
- [4a3a54a](https://github.com/uber/mockolo/pull/262/commits/4a3a54a530f677409ac34d1d2e2e19006324f35b)